Bootcamp.AutomatedBackupService/Worker.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Bootcamp.Data.Enums;
+using Bootcamp.Data.Interfaces;
+using Bootcamp.Data.Models;
+using System.Text;
+
+namespace Bootcamp.AutomatedBackupService
+{
+    internal class Worker
+    {
+        private readonly IEngagementRepository _engagementRepository;
+
+        public Worker(IEngagementRepository engagementRepository)
+        {
+            _engagementRepository = engagementRepository;
+        }
+
+       internal async Task StartProcess()
+        {
+            var backupFrequency = _engagementRepository.GetEngagementBackupFrequency();
+
+            if (backupFrequency == null) return;
+
+            int backupFreqHours = (int)(BackupFrequency)Enum.Parse(typeof(BackupFrequency), backupFrequency);
+
+            await BackupEngagements(backupFreqHours);
+        }
+
+        async Task BackupEngagements(int backupFreqHours)
+        {
+            Console.WriteLine("Inside here");
```

Comment:
remove comment

Reply:
fixed in this commit [79104cf](https://github.com/pushkarcopilot/Bootcamp-Training-NV-API/pull/6/commits/79104cf0c6d4881f6a93181b9375ba3b1274019f)